tests/unit/amo/components/TestRating.js

```diff
@@ -69,411 +95,333 @@ describe(__filename, () => {
     });
 
     it('renders read-only selected stars', () => {
-      const root = render({ rating: 3, readOnly: true });
+      render({ rating: 3, readOnly: true });
 
       // Make sure only the first 3 stars are selected.
       [1, 2, 3].forEach((rating) => {
-        expect(getStar({ root, rating })).toHaveClassName(
-          'Rating-selected-star',
-        );
+        expect(getStar({ rating })).toHaveClass('Rating-selected-star');
       });
       [4, 5].forEach((rating) => {
-        expect(getStar({ root, rating })).not.toHaveClassName(
-          'Rating-selected-star',
-        );
-      });
-    });
-
-    it('renders an accessible description for read-only ratings', () => {
-      const root = render({ rating: 3, readOnly: true });
-
-      [1, 2, 3, 4, 5].forEach((rating) => {
-        expect(getStar({ root, rating })).toHaveProp(
-          'title',
-          'Rated 3 out of 5',
-        );
+        expect(getStar({ rating })).not.toHaveClass('Rating-selected-star');
       });
     });
 
     it("only passes the selected and yellow prop to the IconStar component when it's not readOnly", () => {
-      const root = render({ readOnly: false });
-
-      const star = root.find(IconStar).at(0);
-
-      expect(star).toHaveProp('selected');
-      expect(star).not.toHaveProp('half');
-    });
-
-    it("passes several props to the IconStar component when it's readOnly", () => {
-      const root = render({ readOnly: true });
-
-      const star = root.find(IconStar).at(0);
+      render({ readOnly: false });
+      const firstRating = screen.getByClassName('Rating-rating-1');
 
-      expect(star).toHaveProp('selected');
-      expect(star).toHaveProp('half');
-      expect(star).toHaveProp('yellow');
-      expect(star).toHaveProp('readOnly', true);
+      // This is the result of "selected".
+      expect(within(firstRating).getByTagName('path')).toHaveAttribute(
+        'd',
+        getSvgPath(OPEN_STYLE),
+      );
+      expect(within(firstRating).getByTagName('g')).toHaveAttribute(
+        'fill',
+        photon.YELLOW_50,
+      );
     });
-  });
 
-  describe('rating counts', () => {
-    it('renders the average rating', () => {
-      const rating = 3.5;
-      const root = render({ rating, readOnly: true });
-
-      expect(root).toHaveProp('title', `Rated ${rating} out of 5`);
-    });
+    it("passes expected props to the IconStar component when it's readOnly", () => {
+      render({ readOnly: true });
+      const firstRating = screen.getByClassName('Rating-rating-1');
 
-    it('localizes average rating', () => {
-      const i18n = fakeI18n({ lang: 'de' });
-      expect(render({ rating: 3.5, i18n, readOnly: true }).text()).toContain(
-        '3,5',
+      // This is the result of the default props from readOnly.
+      expect(within(firstRating).getByTagName('path')).toHaveAttribute(
+        'd',
+        getSvgPath(DIM_CLOSED_STYLE),
       );
     });
+  });
 
-    it('renders empty ratings', () => {
```

Comment:
what happened to this test?

Reply:
This does the same thing as [`it('renders correct and accessible titles with a null rating when read-only'`](https://github.com/mozilla/addons-frontend/pull/11367/files#diff-2f6e8a93d203a649be59e8af28b456630e51657dbb6383635f3c9841e17d737bR347)